### ISSUE TITLE
Add a "Select All" button next to each filter

### ISSUE THREAD

**Issue by marco-c** [mozilla/bugbug#2109]
Right now users have to manually select all the items, it'd be nice to add a small "Select All" button next to each filter which has multiple selectable options.

**Comment by Aayushap1** [comment]
I would like to work on this issue please assign this issue to me.

**Comment by marco-c** [comment]
@Aayushap1 feel free to work on this. An issue is assigned automatically when you open a PR linked to it (read https://github.com/mozilla/bugbug/issues/1092).

**Comment by Aayush-hub** [comment]
@marco-c Working on it!

**Comment by webbdays** [comment]
> Right now users have to manually select all the items, it'd be nice to add a small "Select All" button next to each filter which has multiple selectable options.

How to view bugbug ui in the browser for the development purpose i.e I want the changes in code to reflect in bugbug ui website browser.

**Comment by marco-c** [comment]
```
cd ui/changes
npm run dev
```

**Comment by webbdays** [comment]
Ok

**Comment by webbdays** [comment]
i am getting this error :
[snowpack] Cannot find module '@js-temporal/polyfill/lib/index.mjs'

**Comment by webbdays** [comment]
I get to know that the error is because temporal not found.

`npm install temporal`

**Comment by marco-c** [comment]
Yes, first of all you will need to install dependencies (run `npm install` after `cd ui/changes`).
